borgmatic/borg/export_tar.py

```diff
@@ -47,7 +47,11 @@ def export_tar_archive(
         + (('--tar-filter', tar_filter) if tar_filter else ())
         + (('--strip-components', str(strip_components)) if strip_components else ())
         + flags.make_repository_archive_flags(
-            repository if ':' in repository else os.path.abspath(repository),
+            os.path.abspath(repository)
+            if ':' not in repository
+            else os.path.abspath(repository[7:])
```

Comment:
Here's a way to do this last part that avoids a magic number (or at least uses a slightly less magic one):

```python
os.path.abspath(repository.partition('file://')[-1])
```

An alternative:

```python
os.path.abspath(repository[len('file://'):])
```

Reply:
yeah, I used the second suggestion before I used the magic number, but got some interesting conflicts between black and flake8 😅. Will share on IRC and try the first suggestion.